packages/@aws-cdk/aws-events-targets/lib/sqs.ts

```diff
@@ -58,15 +61,18 @@ export class SqsQueue implements events.IRuleTarget {
       })
     );
 
+    let fifoParameters = {};
+    if (!!this.props.messageGroupId) {
+      fifoParameters = { sqsParameters: { messageGroupId: this.props.messageGroupId } };
+    }
+
     const result: events.RuleTargetConfig = {
       id: '',
       arn: this.queue.queueArn,
       input: this.props.message,
       targetResource: this.queue,
+      ...fifoParameters,
```

Comment:
Style: I would just write this as:

```ts
return {
  id: '',
  arn: this.queue.queueArn,
  input: this.props.message,
  targetResource: this.queue,
  sqsParameters: this.props.messageGroupId ? { messageGroupId: this.props.messageGroupId } : undefined
};
```

Reply:
Do you have anything against the general use of the spread operator, or is it just this case that you don't like?